quickstart.md

```diff
@@ -133,22 +133,23 @@ kubectl get runtimeclass
 Output:
 ```
 NAME            HANDLER         AGE
-kata            kata            9m55s
+kata            kata-qemu       9m55s
 kata-clh        kata-clh        9m55s
-kata-clh-tdx    kata-clh-tdx    9m55s
 kata-qemu       kata-qemu       9m55s
-kata-qemu-tdx   kata-qemu-tdx   9m55s
 kata-qemu-sev   kata-qemu-sev   9m55s
+kata-qemu-snp   kata-qemu-snp   9m55s
+kata-qemu-tdx   kata-qemu-tdx   9m55s
 ```
 
 Details on each of the runtime classes:
 
 - *kata* - standard kata runtime using the QEMU hypervisor including all CoCo building blocks for a non CC HW
```

Comment:
This has to be modified.

kata provides only a "link" for kata-qemu, but this is not using all CoCo building blocks for a  non TEE HW.
That's on the `kata-coco-qemu-dev` one.

Reply:
updated